service/gcsutils/remotefs/utils.go

```diff
@@ -146,3 +147,9 @@ func WriteTarOptions(w io.Writer, opts *archive.TarOptions) error {
 
 	return nil
 }
+
+// Sync syncs the filesystem so that all writes buffered in the kernel are flushed
+// to disk.
+func Sync() error {
+	return exec.Command("sync").Run()
```

Comment:
This is a pretty heavy hammer. Can't we implement this just to fsync the files that have been modified?